okhttp/src/main/java/io/grpc/transport/okhttp/OkHttpChannelBuilder.java

```diff
@@ -94,24 +97,36 @@ public OkHttpChannelBuilder transportExecutor(ExecutorService executor) {
    *
    * <p>Should only used by tests.
    */
-  public void overrideHostForAuthority(String host) {
+  public OkHttpChannelBuilder overrideHostForAuthority(String host) {
     this.host = host;
+    return this;
   }
 
   /**
-   * Provides a SSLSocketFactory to establish a secure connection. By default TLS is not enabled.
+   * Provides a SSLSocketFactory to establish a secure connection.
    */
   public OkHttpChannelBuilder sslSocketFactory(SSLSocketFactory factory) {
     this.sslSocketFactory = factory;
     return this;
   }
 
+  /**
+   * For secure connection, provides a ConnectionSpec to specify Cipher suite and
+   * TLS versions.
+   *
+   * <p>By default OkHttpTlsUpgrader.DEFAULT_SPEC will be used.
```

Comment:
We don't want to expose our package hack to our users. We should find a better place for DEFAULT_SPEC within io.grpc.transport.okhttp package.

It _could_ go on the OkHttpChannelBuilder. I wonder if there is a better place maybe.